### PR TITLE
Vimdoc

### PR DESCRIPTION
**Pull request by sim590**
I thought that vim documentation wouldn't be too much since it's possible to forget about the mappings. I've made the mappings from #8 in seperate in case they would be changed or whatever.

### CODE REVIEW

**Comment by christoomey** [comment]
Again, thanks for great work @sim590! This looks good to go as is, but do you mind rebasing onto master and squashing down to a single commit?

File: doc/quicklink.txt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+<C-k>                   When in 'visual' mode searches for the selected
+                        expression and opens an interactive window in which
+                        the first four search results of the expression have
+                        been listed.
+
+gx                      Open a markdown formated link under the cursor.
+                        "NOTE": The default behavior of `gx` when your cursor
+                        is over a URL will still work and is unchanged.
+
```
Comment (christoomey) [inline doc/quicklink.txt]: Leading whitespace on this line. Can you clear that up?

**Comment by sim590** [comment]
I've squashed the commits down into a single one in 41e93eb.

**Comment by christoomey** [comment]
Thanks! Merged into master as cbbce4855d8a64ddfedc1ab07724ea497601929a.

**Pull request closed without merging**